docs/developer-tools/errors.md

```diff
@@ -16,7 +16,7 @@ Find details on common error codes and how to resolve them below.
 Error Code | Meaning
 ---------- | -------
 200 | OK -- Success.
-201 | Accepted -- Your request is successfully accepted.
+201 | Created -- Your request is successfully led to creation of resource.
```

Comment:
Please check the grammar. @amritesh-singh. It should read:

"Your request successfully led to the creation of a resource".